modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/FirebaseAnalyticsWrapper.kt

```diff
@@ -0,0 +1,14 @@
+package au.com.shiftyjelly.pocketcasts.analytics
+
+import android.os.Bundle
+import com.google.firebase.analytics.FirebaseAnalytics
+import com.google.firebase.analytics.ktx.logEvent
+
+/**
+ * A wrapper around Firebase Analytics to allow for mocking as the class is final.
+ */
+open class FirebaseAnalyticsWrapper(private val firebaseAnalytics: FirebaseAnalytics?) {
```

Comment:
There's not need to make `FirebaseAnalytics` nullable. `FirebaseAnalytics.getInstance()` is annotated with `@NonNull`.

Reply:
I had a bit of trouble mocking this class. If I didn't make it nullable I got the following error, but now it's working so I will remove the question marks.
```
Mockito cannot mock this class: class au.com.shiftyjelly.pocketcasts.analytics.FirebaseAnalyticsWrapper.
```